static_src/stores/login_store.js

```diff
@@ -0,0 +1,33 @@
+
+import BaseStore from './base_store.js';
+import { loginActionTypes } from '../constants.js';
+
+
+class LoginStore extends BaseStore {
+  constructor() {
+    super();
+    this.subscribe(() => this._registerToActions.bind(this))
+    this._isAuthenticated = false;
+  }
+
+  _registerToActions(action) {
+    switch(action.type) {
+      case loginActionTypes.RECEIVED_STATUS:
+        this._isAuthenticated = action.status;
+        this.emitChange();
+        break;
+
+      default:
+        break;
+    }
+  }
+
+  isLoggedIn() {
+    return !!this._isAuthenticated;
```

Comment:
It's a javascript way to cast to a boolean. So the first `!` creates a boolean but negates it, the second one sets it to it's real value.

```js
var k = 's';
!s === false; //true
!!s === true; //true
```